README.md

```diff
@@ -14,16 +26,52 @@ its [Code of Conduct](./CODE_OF_CONDUCT.md) for all its projects and events.
 Please read and uphold this code-of-conduct while participating in Kinvolk
 projects and events.
 
-## Email and Chat
+## Projects
+
+There are many repos in the [Kinvolk](https://github.com/kinvolk/) org, though here is the highlights of projects.
+
+### Flatcar Container Linux
+
+- [Flatcar repositories](https://github.com/search?q=org%3Akinvolk+topic%3Aflatcar&type=repositories)
+- [Flatcar docs](https://kinvolk.io/docs/flatcar-container-linux/latest)
+- [Flatcar issue tracker](https://github.com/kinvolk/flatcar/issues)
+- [contributing to Flatcar](https://github.com/kinvolk/Flatcar#contributing-to-flatcar-container-linux)
+- chat: [#flatcar](irc://irc.freenode.org:6667/#flatcar) IRC channel on freenode.org
+- mailing-list: [flatcar-user](https://groups.google.com/g/flatcar-linux-user)
```

Comment:
Nitpick: I'd not repeat the name of the project since the context is clear, but it's done now so :shrug: .

Reply:
I debated about that too :thinking: